ios/RCTCameraManager.m

```diff
@@ -574,6 +579,11 @@ -(void)captureVideo:(NSInteger)target options:(NSDictionary *)options resolve:(R
     [self initializeCaptureSessionInput:AVMediaTypeAudio];
   }
 
+//  if ([options valueForKey:@"quality"]) {
```

Comment:
thanks for the comments @pjcabrera . I understand this things don't make much sense - this is still WIP and I was just making sure my code was not the one breaking the project